sdk/daml-lf/scenario-interpreter/src/main/scala/com/digitalasset/daml/lf/scenario/ScenarioRunner.scala

```diff
@@ -425,16 +425,13 @@ private[lf] object ScenarioRunner {
       commands: SExpr,
       location: Option[Location],
       seed: crypto.Hash,
+      packageResolution: Map[PackageName, PackageId] = Map.empty,
       traceLog: TraceLog = Speedy.Machine.newTraceLog,
       warningLog: WarningLog = Speedy.Machine.newWarningLog,
       doEnrichment: Boolean = true,
   )(implicit loggingContext: LoggingContext): SubmissionResult[R] = {
     val ledgerMachine = Speedy.UpdateMachine(
-      packageResolution = Map(
-        PackageName.assertFromString("-dummy-package-name-") -> PackageId.assertFromString(
-          "-homePackageId-"
-        )
-      ),
```

Comment:
I'm stumped here, it looks like this was necessary given that the integration test `Interface.daml` now fails with `CRASH: unexpected exception com.digitalasset.daml.lf.speedy.SError$SErrorCrash: CRASH (com.digitalasset.daml.lf.speedy.SBuiltinFun$): Could not resolve packageName to packageId: -dummy-package-name- when running continuation of question NeedContract`, but I couldn't find an equivalent `packageResolution` on the `main` branch


full error:
```
      Check diagnostics:                                                                                       FAIL
        File:    
          /home/vsts/.cache/bazel/_bazel_vsts/bbff76d6420e88f7a355eecb81543c75/sandbox/linux-sandbox/2324/execroot/com_github_digital_asset_daml/bazel-out/k8-opt/bin/compiler/damlc/tests/integration-v21.runfiles/com_github_digital_asset_daml/compiler/damlc/tests/daml-test-files/Interface.daml
        Hidden:   no
        Range:    1:1-2:1
        Source:   Daml-LF typechecker
        Severity: DsWarning
        Message: 
        warning while type checking module Interface:
          This package defines both interfaces and templates. This may make this package and its dependents not upgradeable.
          
          It is recommended that interfaces are defined in their own package separate from their implementations.
          
          Ignore this error message with the --warn-bad-interface-instances=yes flag.
        File:    
          /home/vsts/.cache/bazel/_bazel_vsts/bbff76d6420e88f7a355eecb81543c75/sandbox/linux-sandbox/2324/execroot/com_github_digital_asset_daml/bazel-out/k8-opt/bin/compiler/damlc/tests/integration-v21.runfiles/com_github_digital_asset_daml/compiler/damlc/tests/daml-test-files/Interface.daml
        Hidden:   no
        Range:    17:11-17:16
        Source:   Daml-LF typechecker
        Severity: DsWarning
        Message: 
        warning while type checking interface Interface.Token :
          The interface Token was defined in this package and implemented in this package by the following templates:
          
          'Asset'
          
          This may make this package and its dependents not upgradeable.
          
          It is recommended that interfaces are defined in their own package separate from their implementations.
          
          Ignore this error message with the --warn-bad-interface-instances=yes flag.
        File:    
          /home/vsts/.cache/bazel/_bazel_vsts/bbff76d6420e88f7a355eecb81543c75/sandbox/linux-sandbox/2324/execroot/com_github_digital_asset_daml/bazel-out/k8-opt/bin/compiler/damlc/tests/integration-v21.runfiles/com_github_digital_asset_daml/compiler/damlc/tests/daml-test-files/Interface.daml
        Hidden:   no
        Range:    83:1-83:5
        Source:   Script
        Severity: DsError
        Message: 
        Script execution failed on commit at Interface:91:29:
          CRASH: unexpected exception com.digitalasset.daml.lf.speedy.SError$SErrorCrash: CRASH (com.digitalasset.daml.lf.speedy.SBuiltinFun$): Could not resolve packageName to packageId: -dummy-package-name- when running continuation of question NeedContract
        
        Ledger time: 1970-01-01T00:00:00Z
        
        Partial transaction:
        
        Committed transactions: 
          TX 0 1970-01-01T00:00:00Z (Interface:85:16)
          #0:0
          │   disclosed to (since): 'Alice' (0)
          └─> 'Alice' creates Interface:Asset
                      with
                        issuer = 'Alice'; owner = 'Alice'; amount = 15
        Wrong number of diagnostics, expected 2, but got 3
        
        Use -p '/Tests for Daml-LF 2.1.Interface.daml.Check diagnostics/' to rerun this test only.
```